src/sentry/monitors/logic/mark_failed.py

```diff
@@ -301,7 +300,7 @@ def create_issue_platform_occurrence(
         "contexts": {"monitor": get_monitor_environment_context(monitor_env)},
         "environment": monitor_env.environment.name,
         "event_id": occurrence.event_id,
-        "fingerprint": fingerprint
+        "fingerprint": [fingerprint]
```

Comment:
Why did this have to change?

Reply:
separate fix. can break it out. fingerprint was invalid only noticed it during testing